bin/compute-adam-jars.sh

```diff
@@ -22,8 +22,8 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 
 CLASSPATH=$("$ADAM_REPO"/bin/compute-adam-classpath.sh)
 
-# list of jars to ship with spark; trim off the first and last from the CLASSPATH
-# TODO: brittle? assumes appassembler always puts the $BASE/etc first and the CLI jar last
+# list of jars to ship with spark; trim off the first from the CLASSPATH
+# TODO: brittle? assumes appassembler always puts the $BASE/etc first
```

Comment:
This comment was never updated along with the relevant change.  Speaking of which, will distributing the cli jar with the `--jars` option stay kosher?